core/src/main/scala/kafka/coordinator/GroupMetadataManager.scala

```diff
@@ -659,10 +660,10 @@ class GroupMetadataManager(val brokerId: Int,
       config.offsetsTopicNumPartitions
   }
 
-  private def getMessageFormatVersionAndTimestamp(partition: Int): (Byte, Long) = {
+  private def getMessageFormatVersionAndTimestampIfReplicaIsLocal(partition: Int): (Byte, Long) = {
```

Comment:
I was thinking the original name is already clear enough if we have a short comment explaining the error case. I'm OK with the additional verbosity, I just thought it might be overkill (the name is 51 characters!).